compiler/src/main/java/com/squareup/anvil/compiler/AnvilComponentRegistrar.kt

```diff
@@ -99,4 +82,9 @@ class AnvilComponentRegistrar : ComponentRegistrar {
       .getExtensionPoint(AnalysisHandlerExtension.extensionPointName)
       .registerExtension(extension, LoadingOrder.FIRST, project)
   }
+
+  private fun loadCodeGenerators(): List<CodeGenerator> {
+    return ServiceLoader.load(CodeGenerator::class.java, CodeGenerator::class.java.classLoader)
+      .toList()
+  }
```

Comment:
I suggested a different file and didn't realize where it's used 🤦

Reply:
I knew what you meant in spirit :)